apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -94,6 +104,7 @@ const CurriculumCatalog = ({
                   scriptId={script_id}
                   isStandAloneUnit={is_standalone_unit}
                   onAssignSuccess={response => handleAssignSuccess(response)}
+                  quickViewDisplayed={isQuickViewDisplayed}
```

Comment:
With the above change, this can be simplified to `quickViewDisplayed={getQuickViewState}`